mkdocs/docs/HPC/openFOAM.md

```diff
@@ -73,15 +73,23 @@ modules. To get an overview of the available modules, run
 
 ```
 $ module avail OpenFOAM
------------------- /apps/gent/CO7/sandybridge/modules/all ------------------
-   OpenFOAM/v1712-foss-2017b     OpenFOAM/4.1-intel-2017a
-   OpenFOAM/v1712-intel-2017b    OpenFOAM/5.0-intel-2017a
-   OpenFOAM/2.2.2-intel-2017a    OpenFOAM/5.0-intel-2017b
-   OpenFOAM/2.2.2-intel-2018a    OpenFOAM/5.0-20180108-foss-2018a
-   OpenFOAM/2.3.1-intel-2017a    OpenFOAM/5.0-20180108-intel-2017b
-   OpenFOAM/2.4.0-intel-2017a    OpenFOAM/5.0-20180108-intel-2018a
-   OpenFOAM/3.0.1-intel-2016b    OpenFOAM/6-intel-2018a            (D)
-   OpenFOAM/4.0-intel-2016b
+
+------------------------------- /apps/gent/RHEL8/cascadelake-ib/modules/all --------------------------------
+   OpenFOAM-Extend/4.0-intel-2019b-Python-2.7.16                 OpenFOAM/2.3.1-intel-2019b
+   OpenFOAM-Extend/4.1-20191120-intel-2019b-Python-2.7.16        OpenFOAM/5.0-20180606-foss-2019b
+   OpenFOAM-Extend/4.1-20200408-foss-2019b-Python-2.7.16         OpenFOAM/6-foss-2019b
+   OpenFOAM-Extend/4.1-20210705-foss-2023a-Python-2.7.18  (D)    OpenFOAM/7-foss-2019b
+   OpenFOAM/v1906-foss-2019b                                     OpenFOAM/7-foss-2019b-20200508
+   OpenFOAM/v1912-foss-2019b                                     OpenFOAM/8-foss-2020a
+   OpenFOAM/v2006-foss-2020a                                     OpenFOAM/8-foss-2020b
+   OpenFOAM/v2012-foss-2020a                                     OpenFOAM/9-foss-2021a
```

Comment:
@lbarraga Do the `module avail OpenFOAM` when `cluster/shinx` is loaded, that will trim this down a load to modules that will be available on RHEL9